pricejar/base/models.py

```diff
@@ -2,6 +2,17 @@
 from tkinter import CASCADE
 from unicodedata import name
 from django.db import models
+from django.contrib.auth.models import User
+
 
 
 # Create your models here.
+
+# User profile
+class UserProfile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
Good work. I might do some changes on this file since I am working on the user model. someone suggested we work with abstract user. Anyways your work is impressive.

Reply:
Thank you